community-operators/postgresql/postgresql.package.yaml

```diff
@@ -1,4 +1,4 @@
 packageName: postgresql
 channels:
-- name: alpha
+- name: stable
```

Comment:
For the record, this value need not match the `maturity` value in the CSV. This change is not required

Reply:
@SamiSousa Should I revert this or leave it be?

Reply:
I recommend reverting it :)

Reply:
@SamiSousa Done 😄 And it will trigger a build, so we shall see how it goes!

Reply:
Could you also revert the change in the upstream's package.yaml as well? 😃

Reply:
@SamiSousa Done!